tests/components/mdx.test.tsx

```
@@ -0,0 +1,47 @@
+import { render, screen, fireEvent, waitFor } from "@testing-library/react";
+import { Callout } from "@/components/mdx/callout";
+import { CustomLink } from "@/components/mdx/link";
+import { CodeBlock } from "@/components/mdx/code-block";
+
+jest.mock("next/image", () => ({
+  __esModule: true,
+  default: (props: any) => <img {...props} />,
```

Biome (2.1.2)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)